internal/pkg/encryption/keys/generated.go

```diff
@@ -0,0 +1,46 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package keys
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/talos-systems/go-smbios/smbios"
+)
+
+// GeneratedKeyHandler generates the key based on current node information
```

Comment:
I think @Ulexus might have a better idea, but something like `DeterminsticKeyHandler` better describes what is going on here.